examples/profile/profile.spec.js

```diff
@@ -0,0 +1,114 @@
+require('dotenv').config();
+
+const {
+  Client,
+  YotiDate,
+} = require('yoti');
+
+const {
+  SandboxProfileClientBuilder,
+  SandboxAgeVerificationBuilder,
+  SandboxAnchorBuilder,
+  TokenRequestBuilder,
+} = require('@getyoti/sdk-sandbox');
+
+const fs = require('fs');
+
+const config = {
+  SANDBOX_CLIENT_SDK_ID: process.env.YOTI_SANDBOX_CLIENT_SDK_ID,
+  PEM: fs.readFileSync(process.env.YOTI_KEY_FILE_PATH, 'utf8'),
+};
+
+describe('Sandbox Example', () => {
+  it('should return a user profile', async () => {
+    const anchors = [
+      new SandboxAnchorBuilder()
+        .withType('SOURCE')
+        .withSubType('')
+        .withTimestamp(YotiDate.fromDateString('2020-01-01T00:00:00Z'))
+        .withValue('PASSPORT')
+        .build(),
+      new SandboxAnchorBuilder()
+        .withType('VERIFIER')
+        .withSubType('')
+        .withTimestamp(YotiDate.fromDateString('2020-01-01T00:00:00Z'))
+        .withValue('YOTI_ADMIN')
+        .build(),
+    ];
+
+    const ageVerification = new SandboxAgeVerificationBuilder()
+      .withDateOfBirthString('1980-01-01')
+      .withAgeOver(18)
+      .build();
+
+    const tokenRequest = new TokenRequestBuilder()
+      .withRememberMeId('Some Remember Me ID')
+      .withGivenNames('Some Given Names', anchors)
+      .withFamilyName('Some Family Name', anchors)
+      .withFullName('Some Full Name', anchors)
+      .withDateOfBirthString('1980-01-02', anchors)
```

Comment:
Can we either change this value, or the ageVerification value to be the same, to avoid confusion
```suggestion
      .withDateOfBirthString('1980-01-01', anchors)
```

Reply:
Fixed in c1cf35c